styles/angular-patternfly.less

```diff
@@ -1,3 +1,4 @@
+@import "../node_modules/patternfly/node_modules/bootstrap/less/variables.less";
```

Comment:
What?!  Didn't you stop me from adding bootstrap variables?  #543

Reply:
LOL, right... I will remove.